.github/workflows/build.yml

```diff
@@ -4,17 +4,24 @@ on: [push, pull_request]
 
 jobs:
   build:
+    strategy:
+      matrix:
+        java: [ 17-ubuntu, 21-ubuntu ]
     runs-on: ubuntu-22.04
     container:
-      image: eclipse-temurin:17-jdk
+      image: mcr.microsoft.com/openjdk/jdk:${{ matrix.java }}
```

Comment:
Why Microsoft java specifically?

Reply:
Technically because that's the one used in vanilla launcher.
But I guess most of our users are using multimc based launchers.